aws-go-lambda/Makefile

```diff
@@ -0,0 +1,4 @@
+build::
```

Comment:
It may be inevitable - but having a Makefile here means this may not work well on Windows. There are some notes on doing this in a cross platform compatible way in https://github.com/aws/aws-lambda-go/blob/master/README.md#for-developers-on-windows.